app/search/server/events/events.js

```diff
@@ -22,45 +25,57 @@ const eventService = new EventService();
 /**
  * Listen to message changes via Hooks
  */
-callbacks.add('afterSaveMessage', function(m) {
+function afterSaveMessage(m) {
 	eventService.promoteEvent('message.save', m._id, m);
 	return m;
-}, callbacks.priority.MEDIUM, 'search-events');
+}
 
-callbacks.add('afterDeleteMessage', function(m) {
+function afterDeleteMessage(m) {
 	eventService.promoteEvent('message.delete', m._id);
 	return m;
-}, callbacks.priority.MEDIUM, 'search-events-delete');
+}
 
 /**
  * Listen to user and room changes via cursor
  */
-
-
-Users.on('change', ({ clientAction, id, data }) => {
+function onUsersChange({ clientAction, id, data }) {
 	switch (clientAction) {
 		case 'updated':
 		case 'inserted':
-			const user = data || Users.findOneById(id);
+			const user = data ?? Users.findOneById(id);
 			eventService.promoteEvent('user.save', id, user);
 			break;
 
 		case 'removed':
 			eventService.promoteEvent('user.delete', id);
 			break;
 	}
-});
+}
 
-Rooms.on('change', ({ clientAction, id, data }) => {
+function onRoomsChange({ clientAction, id, data }) {
 	switch (clientAction) {
 		case 'updated':
 		case 'inserted':
-			const room = data || Rooms.findOneById(id);
+			const room = data ?? Rooms.findOneById(id);
 			eventService.promoteEvent('room.save', id, room);
 			break;
 
 		case 'removed':
 			eventService.promoteEvent('room.delete', id);
 			break;
 	}
-});
+}
+
+settings.get('Search.Provider', _.debounce((key, value) => {
```

Comment:
why isn't "lint" complaining about `key` and `value` variables not being used? 🤔